doc/classes/@GlobalScope.xml

```diff
@@ -2293,10 +2294,10 @@
 			Middle mouse button.
 		</constant>
 		<constant name="MOUSE_BUTTON_WHEEL_UP" value="4" enum="MouseButton">
-			Mouse wheel up.
+			Mouse wheel up button.
```

Comment:
It's technically a discrete event handled like a button, but I'm not sure it can be described as such to end users. When you roll your mouse wheel up/down you don't feel like you're pressing buttons.